types/index.d.ts

```diff
@@ -14,4 +14,4 @@ export * from './queries'
 export * from './query-helpers'
 export * from './suggestions'
 export * from './wait-for'
-export type * from '../src/user-event'
+export type {keyboardKey} from '../src/user-event/index'
```

Comment:
```suggestion
export type * from '../src/user-event/index'
```

Reply:
<img width="439" alt="image" src="https://user-images.githubusercontent.com/1443499/171290824-f9b2e101-b64f-4099-b77b-5f09754a04f6.png">

Reply:
`export type * from` isn't something supported or implemented in TypeScript.

Reply:
Oh wait, really?

That's super unfortunate. I swore it would've (and should've) failed tests. Sorry for merging early! I'll go back and update this file to fix the issue.

Reply:
I'm pretty sure those files aren't checked because it would make TS throw errors (for example for the `Array<string, Buffer>` as well)

Reply:
Yeah, that's really weird. WebStorm (my IDE) doesn't pick up this error and I can't get VSCode or `tsc` to, either.

What version of TypeScript and/or other environment settings (OS, Node version) are you using?

Reply:
Hmm, yeah, makes sense. I'll see what I can do about fixing that

Reply:
I'm using VSCode, but indeed it's weird because I can only see those errors when consuming your package and disabling `skipLibCheck`, then if I go within node_modules/cli-testing-library the errors appear. 🤔

Reply:
I has something to do with `"extends": "./node_modules/kcd-scripts/shared-tsconfig.json",` in your root `tsconfig.json`, if you delete it you'll see the errors.

Reply:
Figured it out - it's because `shared-tsconfig.json` has `skipLibCheck` set to `true`, which we don't want.

Removing it seems to solve the problem. Thanks a ton for your help!

Reply:
These issues should be solved as of e947a8e8ded895f4df212be8b84d64673ac42794 in `main`.